src/Raven.Server/Web/Studio/StudioTasksHandler.cs

```diff
@@ -50,19 +51,43 @@ public Task FullDataDirectory()
             {
                 if (PathUtil.IsSubDirectory(result, ServerStore.Configuration.Core.DataDirectory.FullPath) == false)
                 {
-                    result = $"The administrator has restricted databases to be created only under the {RavenConfiguration.GetKey(x => x.Core.DataDirectory)} directory: '{ServerStore.Configuration.Core.DataDirectory.FullPath}'.";
+                    result =
+                        $"The administrator has restricted databases to be created only under the {RavenConfiguration.GetKey(x => x.Core.DataDirectory)} directory: '{ServerStore.Configuration.Core.DataDirectory.FullPath}'.";
                 }
             }
-            using (ServerStore.ContextPool.AllocateOperationContext(out JsonOperationContext context))
-            using (var writer = new BlittableJsonTextWriter(context, ResponseBodyStream()))
+
+            var drivesInfo = PlatformDetails.RunningOnPosix ? DriveInfo.GetDrives() : null;
+            var driveInfo = DiskSpaceChecker.GetDriveInfo(result, drivesInfo);
```

Comment:
Why are we passing the `drivesInfo`?

Reply:
I don't follow?

Reply:
We need the DrivesInfo in Linux to find the root mount.
The reason for that is that the provided URL might not exists.
For example: /mnt/Data/NewDatabase. The root mount is: /mnt/Data/.
Running the DiskSpaceChecker on a folder that doesn't exist doesn't provide any result.

https://github.com/ravendb/ravendb/blob/v4.1/src/Sparrow/Utils/DiskSpaceChecker.cs#L141